### PR TITLE
Revert snow day hours

### PR DESCRIPTION
**Pull request by rye**
As the skies are clear and the roads are getting better, it's probably a good idea to revert the two snow day PR's, #2198 and #2201.

### CODE REVIEW

**Review by drewvolz — DISMISSED** [review]

**Comment by drewvolz** [comment]
Not sure if we should leave it as "Van-GO" or "Van-Go"

**Comment by hawkrives** [comment]
Caps

**Comment by hawkrives** [comment]
fixed

**Comment by drewvolz** [comment]
Might as well go the whole way and make the title correct with "Van-GO!"

**Comment by rye** [comment]
Well, I just reverted the commit.  Also, that synopsis hadn't been touched since Thanksgiving.

File: data/transportation/van-go.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: Van-GO!
 category: Bus
 url: http://van-go.info/schedule/
-synopsis: Van-GO is stopped due to snow. Van service that operates Monday - Friday from 7 a.m. to 6 p.m. Business use only.
+synopsis: Van service that operates Monday - Friday from 7 a.m. to 6 p.m. Business use only.
 description:
-  Van-GO is stopped today due to snow. Van service that operates Monday - Friday from 7 a.m. to 6 p.m. Business use only.
+  Van service that operates Monday - Friday from 7 a.m. to 6 p.m. Business use
+  only.
```
Comment (drewvolz) [inline data/transportation/van-go.yaml:7]: should the word "only" be on a new line?

Comment (hawkrives) [reply data/transportation/van-go.yaml:7]: That's how it wraps

Comment (drewvolz) [reply data/transportation/van-go.yaml:7]: It wouldn't have a line number, would it?

Comment (hawkrives) [reply data/transportation/van-go.yaml:7]: Correct. I wrap these at 80 columns.

Comment (drewvolz) [reply data/transportation/van-go.yaml:7]: Alright :+1:

**Review by drewvolz — APPROVED** [review]

**Pull request merged**